### PR TITLE
reverting profile util temporary

### PR DESCRIPTION
**Pull request by xetys**
until some better solution is found, to prevert cloud config failures in production

### CODE REVIEW

**Comment by brunnels** [comment]
You should not revert the ```public static String[] getActiveProfiles(Environment env)``` method in ```_DefaultProfileUtil.java``` so you will not need to revert the changes in _ProfileInfoResource.java.

File: generators/server/templates/src/main/java/package/config/_DefaultProfileUtil.java [hunk 1]
```diff
@@ -26,18 +55,23 @@ public static void addDefaultProfile(SpringApplication app) {
...
         app.setDefaultProperties(defProperties);
     }
 
     /**
-     * Get the profiles that are applied else get default profiles.
+     * Load application.yml from classpath.
+     *
+     * @return the YAML Properties
      */
-    public static String[] getActiveProfiles(Environment env) {
```
Comment (deepu105) [inline generators/server/templates/src/main/java/package/config/_DefaultProfileUtil.java]: no need to revert this method.

File: generators/server/templates/src/main/java/package/web/rest/_ProfileInfoResource.java [hunk 2]
```diff
@@ -1,18 +1,18 @@
 package <%=packageName%>.web.rest;
 
-import <%=packageName%>.config.DefaultProfileUtil;
```
Comment (deepu105) [inline generators/server/templates/src/main/java/package/web/rest/_ProfileInfoResource.java]: no need to revert this class as well

**Comment by deepu105** [comment]
Closing in favor of https://github.com/jhipster/generator-jhipster/pull/4184#issuecomment-248973925

**Pull request closed without merging**